src/Core/Query/Helper.php

```diff
@@ -161,11 +161,8 @@ public function formatDate($input)
                     $input = false;
                 }
                 break;
-            // any other input formats can be added in additional cases here...
-            // case $input instanceof Zend_Date:
```

Comment:
StyleCI wanted to indent this one more stop. Looks off to have `// case` line up with `break;`. It's safe to assume that contributors know how to add a case without these instructions. 😉